docs/src/main/asciidoc/application-lifecycle-events-guide.adoc

```diff
@@ -94,6 +94,14 @@ TIP: The events are also called in _dev mode_ between each redeployment.
 
 NOTE: The methods can access injected beans. Check the {quickstarts-blob-url}/application-lifecycle-events/src/main/java/org/acme/events/AppLifecycleBean.java[AppLifecycleBean.java] class for details.
 
+=== What is the difference from `@Initialized(ApplicationScoped.class)` and `@Destroyed(ApplicationScoped.class)`
+
+In the JVM mode, there is no real difference, except that `StartupEvent` is always fired *after* `@Initialized(ApplicationScoped.class)` and `ShutdownEvent` is fired *before* `@Destroyed(ApplicationScoped.class)`. 
+For a native executable build, however, `@Initialized(ApplicationScoped.class)` is fired as *part of the native build process*, whereas `StartupEvent` is fired when the native image is executed.
+See link:extension-authors-guide.html#bootstrap-three-phases[Three Phases of Bootstrap and Quarkus Philosophy] for more details.
```

Comment:
Not strictly mandatory but better use a cross reference so that we get a warning if it's broken. See https://asciidoctor.org/docs/user-manual/#internal-cross-references .

Reply:
Do cross references work accross multiple files?

Reply:
Sorry, I gave you the wrong link: https://asciidoctor.org/docs/user-manual/#inter-document-cross-references

What I'm unsure is if there is anything checked in this case.

Reply:
Done.